### ISSUE TITLE
style: add linting

### ISSUE THREAD

**Issue by madiedgar** [legesher/language-legesher-python#4]
## Linting 

Being able to have formatted code that is aligned with the style guide will be important in the future. However, this might take additional time to make sure the linters apply to "Legesher" grammar and not just "Python" or "Javascript" grammar. 

This quote from [stack overflow](https://stackoverflow.com/questions/48363647/editorconfig-vs-eslint-vs-prettier-is-it-worthwhile-to-use-them-all) gives a good explanation of the different attributes we will touch on:
>**EditorConfig**: This helps your editor produce code that looks like your style guide as you go. While this isn't strictly necessary in order to achieve your goals, it's nice if you're always looking at code that follows the same coding styles. Otherwise if you don't have EditorConfig, as you're typing your editor will auto-format differently to the rest of the code base, which is confusing. Of course if you've set up prettier it'll fix it before it goes into your code base, but still, why would you want to look at it in one format while you're writing it and then have it switch when you go to commit? Might as well be consistent.  
>  
>**Prettier**: Automatically formats your code. I like to set it up to do this when I stage my files for a commit, so that it's physically impossible for me to commit code that doesn't match my style guide.  
>  
>**ESLint**: So why would you want a linter too? Because ESLint does more than just style. It picks up when you declare variables you don't use, or reference things that aren't defined, amongst a few other niceties. So while its role diminishes somewhat compared to the days before prettier, it's still useful to have in a project to catch the other errors.

This will eventually need a larger discussion of adopting best coding practices per programming language.  Note: this repository already has coffeescript linting and contains coffeescript + python. But for now, let's start with the following:

- [ ] Confirm all the coffeescript files are properly linted
- [ ] Add linting rules for the python files

**Comment by rajitha1998** [comment]
Hi, how can I contribute for this issue?

**Comment by madiedgar** [comment]
Thanks @rajitha1998 so much for the desire to contribute to this project! We're wanting to create a consistent code styling for this repository so contributing to the project in the future is easier. I can update the issue description to give you a little more context and we can go through it together. Would you like to take this on? 😄

**Comment by rajitha1998** [comment]
yes I would like to take this task :) . can you give more details about this @madiedgar

**Comment by rajitha1998** [comment]
Hi @madiedgar  I thought of using pylint in-order to lint python files. But for this we need to install this python library. I think we should create a `setup.py` for it. The` .pylintrc` is used to override the default configuration which is same as the `coffeelint.json` file. I have added a sample configuration file which has been used by another opensource project(can change it if you want a different configuration). I tried to check whether the `coffeescript` files are properly linted but the `npm readme file` has not enough information on how to run it. Also there website is not working [http://www.coffeelint.org](url) so I was unable to find information on that .
